<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrator</class>
 <widget class="QDialog" name="Administrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>管理员系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>菜单管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="people_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>人员管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commit_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revert_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>540</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>撤销修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加新项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除旧项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询名字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showall_btn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>610</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示全表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>620</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="peopleView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>741</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="cancle_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>23</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>菜名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>23</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="type_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>73</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>73</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>价格</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="menuView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>741</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加新项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="cancle_btn_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>23</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_edit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>23</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identity_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>73</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>身份</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>73</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_btn_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="change_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>340</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showall_btn_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>610</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示全表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除旧项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_btn_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>610</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>620</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>people_btn</zorder>
  <zorder>menu_btn</zorder>
  <zorder>peopleView</zorder>
  <zorder>label</zorder>
  <zorder>commit_btn</zorder>
  <zorder>revert_btn</zorder>
  <zorder>add_btn</zorder>
  <zorder>delete_btn</zorder>
  <zorder>select_btn</zorder>
  <zorder>showall_btn</zorder>
  <zorder>lineEdit</zorder>
  <zorder>menuView</zorder>
  <zorder>frame</zorder>
  <zorder>add_btn_2</zorder>
  <zorder>back</zorder>
  <zorder>frame_2</zorder>
  <zorder>change_btn</zorder>
  <zorder>showall_btn_2</zorder>
  <zorder>delete_btn_2</zorder>
  <zorder>select_btn_2</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
